Nos/Views/Form/NosFormSection.swift

```diff
@@ -32,7 +32,7 @@ struct NosFormSection<Content: View>: View {
                 .cornerRadius(21)
                 .offset(y: 4.5)
                 .shadow(
-                    color: Color(white: 0, opacity: 0.2), 
+                    color: Color.nosFormSectionShadow, 
```

Comment:
This is the same color as `bigActionButtonShadow`, so I'd remove one and name the other more generically so we don't have to duplicate the color. Something like `light-shadow` in the Colors.xcassets.

Reply:
Was not sure how you wanted duplicates handled; now I know. 🙂